src/test/resources/features/WishlistFunctionality.feature

```diff
@@ -10,23 +11,19 @@ Feature: WishLish functionality
     And alert message is closed
     When user clicks on 'wishlist' HeaderButton and move to 'WishList' page
 
-  @aciuvaga
   Scenario: Add a product to WishList
     Then product 'Nokia Lumia 1020' is displayed in the WishList
 
-  @aciuvaga
   Scenario: Change product quantity in the WishList
     When user fills in '2'
     And user press on 'updatecart' Button
     Then updated product quantity is displayed in the WishList
 
-  @aciuvaga
   Scenario: Remove a product from WishList
-    When user press on 'removefromcart' CheckBox
+    When user presses on 'removefromcart' CheckBox
```

Comment:
Please take care to write correct steps, correcponding to step definition:

@When("^user press on '(.*)' (Button|CheckBox)$")
    public void userPressOnButton(String elementName, String extension)